monasca_rsyslog/client.py

```diff
@@ -34,7 +36,7 @@
 class Client(object):
     """Client for interacting with the monasca-log-api."""
     
-    def __init__(self):
+    def __init__(self, interval=10):
```

Comment:
Please could we make this a config variable?

Reply:
Done